cmd/xl-v1-multipart.go

```diff
@@ -34,6 +34,12 @@ func (xl xlObjects) getUploadIDDir(bucket, object, uploadID string) string {
 	return pathJoin(xl.getMultipartSHADir(bucket, object), uploadID)
 }
 
+// getUploadIDLockPath returns the name of the Lock in the form of
+// bucket/object/uploadID. This is useful in debugging when the locks are listed.
```

Comment:
this comment is better: `For locking, the path bucket/object/uploadID is locked instead of multipart-sha256-Dir/uploadID as it is more readable in the list-locks output which helps in debugging.`